metal3-dev/local-bmo.sh

```diff
@@ -67,7 +67,9 @@ oc apply -f $OUTDIR/bmo-deployment-dev.yaml -n openshift-machine-api
 # Set some variables the operator expects to have in order to work
 export OPERATOR_NAME=baremetal-operator
 
-CLUSTER_IP=$(wrap_if_ipv6 $CLUSTER_PROVISIONING_IP)
+oc wait --for=condition=Ready pod -l baremetal.openshift.io/cluster-baremetal-operator=metal3-state --timeout=90s
```

Comment:
I think you want to look at the metal3 pod, metal3-state is the daemonset running the HTTP cache.

Reply:
@stbenjam Hmm, maybe I'm missing something, but this seems to be the label applied to the Deployment that deploys the metal3 services pod?

```
$ oc get deployment metal3 -o json | jq .spec.template.metadata.labels
{
  "baremetal.openshift.io/cluster-baremetal-operator": "metal3-state",
  "k8s-app": "metal3"
}

$ oc get deployment metal3 -o json | jq .spec.template.spec.containers[].name
"metal3-baremetal-operator"
"metal3-mariadb"
"metal3-httpd"
"metal3-ironic-conductor"
"ironic-inspector-ramdisk-logs"
"metal3-ironic-api"
"ironic-deploy-ramdisk-logs"
"metal3-ironic-inspector"
$ oc get pod -l baremetal.openshift.io/cluster-baremetal-operator=metal3-state
NAME                      READY   STATUS    RESTARTS   AGE
metal3-5657dccb9d-jfg74   8/8     Running   0          9d
```

Reply:
Sorry I didn't realize that was the label - the service for the image cache ends up being called metal3-state.